frontend/src/tests/components/Nav/Footer.test.js

```diff
@@ -29,7 +29,7 @@ describe("Footer tests", () => {
         // expect(text.textContent).toEqual('HappierCows is a project of Mattanjah de Vries, Distinguished Professor of Chemistry at UC Santa Barbara. The open source code is available on GitHub.');
     
         const href = screen.getByTestId("github-href");
-        console.log(href.href);
+        //console.log(href.href);
```

Comment:
Please remove this commented out line of code.  It's good to remove unnecessary console.log output, but we should be bold in doing so; really delete it, don't just comment it out.